src/citrine/__version__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.85.3'
```

Comment:
Link to equivalent file in `requests`: https://github.com/psf/requests/blob/master/requests/__version__.py

They put more information in that file that just the version number. We could do something similar in the future, but this provides a pattern we could expand on in the future.